src/index.js

```diff
@@ -15,7 +15,7 @@ export default function livereload (options = { watch: '' }) {
   const snippetSrc = options.clientUrl
     ? JSON.stringify(options.clientUrl)
     : process.env.CODESANDBOX_SSE
-    ? `'//' + (window.location.host.replace(/^([^\.]+)/, "$1-35729")).split(':')[0] + '/livereload.js?snipver=1&port=443'`
+    ? `'//' + (window.location.host.replace(/^([^.]+)-\d+/,"$1").replace(/^([^\.]+)/, "$1"+port)).split(':')[0] + '/livereload.js?snipver=1&port=443'`
```

Comment:
Have you confirmed that this works? I would expect `${port}` instead of `port`

Reply:
I tested the regex in isolation as well ran the included tests.

I tend to avoid mixing capture groups and template literals, but this should also work.
```
`$1${port}`
```

Reply:
There are no tests for the codesandbox integration, so you would have to copy this file and import it in codesandbox and check if it generates the correct script tag.

`port` is defined in the scope of the plugin, but not in the browser, so I would suggest:

```suggestion
    ? `'//' + (window.location.host.replace(/^([^.]+)-\d+/,"$1").replace(/^([^\.]+)/, "$1"+`+port+`)).split(':')[0] + '/livereload.js?snipver=1&port=443'`
```

Or 
```suggestion
    ? `'//' + (window.location.host.replace(/^([^.]+)-\d+/,"$1").replace(/^([^\.]+)/, "$1${port}")).split(':')[0] + '/livereload.js?snipver=1&port=443'`
```

Reply:
Sorry, you're completely right. I had missed the the fact that the regex was only a template. Codesandbox is having a stroke at the moment, but I was able to test it partly earlier. Your suggested change works, except ``"$1${port}"`` needs to be ``"$1-${port}"``.